Content.Server/Weapons/Melee/EnergyShieldSystem.cs

```diff
@@ -0,0 +1,114 @@
+using Content.Server.CombatMode.Disarm;
+using Content.Server.Kitchen.Components;
+using Content.Shared.Interaction;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Item;
+using Content.Shared.Temperature;
+using Content.Shared.Toggleable;
+using Content.Shared.Tools.Components;
+using Robust.Shared.Player;
+
+namespace Content.Shared.Weapons.Melee.EnergyShield;
```

Comment:
```suggestion
namespace Content.Server.Weapons.Melee.EnergyShield;
```

Since the server class is in server this namespace should also be in Server. Sorry for confusion earlier.

Reply:
done